nupic/regions/TPRegion.py

```diff
@@ -506,6 +517,16 @@ def _compute(self, inputs, outputs):
       size = activeLearnCells.shape[0] * activeLearnCells.shape[1]
       outputs['lrnActiveStateT'][:] = activeLearnCells.reshape(size)
 
+    if self.computePredictedActiveCellIndices:
+      # Reshape so we are dealing with 1D arrays
+      activeState = self._tfdr.getActiveState().reshape(-1).astype('float32')
+      predictedState = self._tfdr.getPredictedState().reshape(-1).astype('float32')
+      activeIndices = set(numpy.where(activeState != 0)[0])
+      predictedIndices= set(numpy.where(predictedState != 0)[0])
+      predictedActiveIndices = activeIndices & predictedIndices
+      outputs["activeCellsIndices"] = numpy.array(list(activeIndices), dtype=int)
```

Comment:
You are overriding the array in `output["activeCellsIndices"]` with a copy of `activeIndices`, right? I think overriding it will cause issues, you should just be modifying the existing array.